frontend/src/components/navbar.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from "@tanstack/react-router";
+
+export default function NavBar() {
+  return (
+    <header className="flex items-center justify-between border-b px-8 py-4 shadow-md">
+      <nav>
+        <Link to="/">
+          <Logo />
+        </Link>
+      </nav>
+      <Auth />
+    </header>
+  );
+}
+
+function Auth() {
+  return <p>Auth Placeholder</p>;
+}
```

Comment:
Make it nothing, so that it isn't noticed in the UI. We need to deliver in increments so there shouldn't be sign of missing things.